edx_rest_framework_extensions/auth/jwt/tests/test_decoder.py

```diff
@@ -68,22 +68,24 @@ def test_valid_token_multiple_valid_issuers(self, jwt_issuer):
 
     def test_failure_invalid_issuer(self):
         """
-        Verifies the function logs decode failures, and raises an InvalidTokenError if the token cannot be decoded
+        Verifies the function logs decode failures with invalid issuer,
+        and raises an InvalidTokenError if the token cannot be decoded
         """
 
         # Create tokens using each invalid issuer and attempt to decode them against
         # the valid issuers list, which won't work
         with mock.patch('edx_rest_framework_extensions.auth.jwt.decoder.logger') as patched_log:
             with self.assertRaises(jwt.InvalidTokenError):
                 self.payload['iss'] = 'invalid-issuer'
-                signing_key = 'invalid-secret-key'
```

Comment:
With this invalid secret code returns earlier with invalid InvalidTokenError. I have split the test into two tests and assert the right log msg.